spec/views/welcome/index.html.erb_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe "welcome/index.html.erb", :type => :view do
+  # pending "add some examples to (or delete) #{__FILE__}"
+  describe "welcome.index.html.erb" do 
+    it "show a welcome page " do 
+      render
+      expect(rendered).to include("Welcome to wiki") 
```

Comment:
Well done for testing the welcome page.  Consider testing static pages like this in feature specs rather than view specs.

E.g. `features/welcome_page_spec.rb`, `features/about_page_spec.rb`. 

The reason the feature specs are slightly preferable for testing static pages is that they test that all the various parts of the app that work together to show a static page. i.e. that these parts are working together correctly: `view <---> controller <---> route/URL`